### ISSUE TITLE
Update the disclaimer

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/light-client#2355]
## Description
Currently, the disclaimer is getting reworked. It needs to be updated once it is done.

## Acceptance criteria
- 

## Tasks

Needs to be done first:
- [x] Add a page to the GitHub repo with the Terms & Conditions (https://docs.google.com/document/d/1Ms3qHF54dw8_AhbsulRAkFe9h1FAg0pttPjLXcv7tRI/edit?ts=60096d59#)
- [x] Update the disclaimer (https://docs.google.com/document/d/1Y3IbX6aOGhcTslxG0jgKvA5Rd6WNau9kBBfwgbbnYbk/edit)
    - [x] Update in the dApp -> Link to the terms & conditions on the GitHub repo
    - [x] Update in the SDK -> Link to the terms & conditions on the GitHub repo
    - [x] Update in the Readme -> Link to the terms & conditions on the GitHub repo
- [x] Add a link to the dApp, which links to the terms & conditions on the GitHub repo (below the link to the privacy policy)
~~- [ ] Update the privacy policy of brainbot.li (https://docs.google.com/document/d/1PGzSOdqdnAZY-Lk81ejeHCFojaV0fFdq4H4meiEcDJE/edit#)~~

**Comment by christianbrb** [comment]
Still waiting for a final approval

**Comment by christianbrb** [comment]
@taleldayekh This is ready for implementation now. I have adopted the task list. Let me know if you have got questions.

**Comment by taleldayekh** [comment]
@christianbrb Thank you, I'll add it :)

**Comment by taleldayekh** [comment]
> Update the privacy policy of brainbot.li (https://docs.google.com/document/d/1PGzSOdqdnAZY-Lk81ejeHCFojaV0fFdq4H4meiEcDJE/edit#)

Will be moved to a separate issue in the team repo.
